examples/MagicWeather/build.gradle

```diff
@@ -1,13 +1,16 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 buildscript {
-    ext.kotlin_version = "1.3.72"
+    ext.compile_version = 31
+    ext.kotlin_version = "1.6.21"
     ext.purchases_version = "5.0.0"
+    ext.lifecycle_version = "2.5.0"
+    ext.androidx_core_version = "1.8.0"
```

Comment:
We are not using the common `build.gradle` for the `MagicWeather` app, so I had to duplicate the versions here... We could have tried to import it, but considering that one has things related to the SDK, it felt better to keep it separate.

Reply:
Yeah I think it's because it's not part of the project